### ISSUE TITLE
Ability to bookmark groups of open files

### ISSUE THREAD

**Issue by nickavv** [romeara/corona#5]
If I'm working on one problem which requires a certain set of files open, then I shift gears to another issue with its own set of open files, I should be able to easily get back to the first set when I want to work on that issue again.
My suggested solution is to be able to bookmark/snapshot certain groups of files (possibly even scroll positions), which can then be named. Later the bookmarked group could be selected from a menu to reopen all of those files.

**Comment by ZPak** [comment]
I've wanted the ability to color tabs for the purposes of grouping. Not sure if that use case overlaps or is separate.

**Comment by nickavv** [comment]
To me that would be more useful to group within one context. Like, if I'm working on a frontend feature I could make all the backend java files tabs a certain color, and the js/css/jade file tabs a different color, so I could tell what part of my context was UI and what Java at a glance. but all within one context. This ticket is more for grouping whole different contexts. But I'd love to hear other opinions

**Comment by romeara** [comment]
Discussed with @nickavv - we think a good way to do this would be to allow multiple "views", and switch between them, similar to how OS's do multiple desktops. This would also allow something similar to Eclipse perspectives, where a user can create a "view" tailored to a specific task

**Comment by nickavv** [comment]
One of the use-cases for Perspectives in Eclipse is to rearrange the layout of the views on the screen. I think the arrangement should be saved as part of this feature, but I think that the ability to change the layout and save a layout should be a separate feature.

**Comment by romeara** [comment]
Fair point - being able to create new views, you'll need to be able to start from a thing, which matches up with that as well. Will add to the proposed feature list
